pkg/config/config.go

```diff
@@ -0,0 +1,58 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"gopkg.in/yaml.v2"
+)
+
+type Worker struct {
+	Count int `yaml:"count"`
+}
+
+type General struct {
+	Development bool `yaml:"development"`
+}
+type Config struct {
+	DB      db.DB   `yaml:"db"`
+	Worker  Worker  `yaml:"worker"`
+	General General `yaml:"general"`
+}
+
+func GetConfig(filePath string) (*Config, error) {
+	// parse configuration from yaml file
+	data, err := os.ReadFile(filePath)
+	if err != nil {
+		return nil, err
+	}
+	cfg := &Config{}
+	err = yaml.Unmarshal([]byte(data), cfg)
+	if err != nil {
+		return nil, err
+	}
+	if cfg.DB.Host == "" {
+		return nil, fmt.Errorf("db:host is not set")
```

Comment:
Paths are donated with `.` not `:`, let try to comply with that 
```suggestion
		return nil, fmt.Errorf("config.db.host is not set")
```